Submodels/Parameterizations.py

```
@@ -0,0 +1,769 @@
+                                           axis = 0)  # Construct complete camber curve over length of profile
+            bezier_camber_x = np.concatenate((x_LE_camber, x_TE_camber),
+                                             axis = 0)  # Construct complete array of x-coordinates over length of profile
+            
+            # Calculate the upper and lower surface coordinates from the bezier coordinates
+            upper_x, upper_y, lower_x, lower_y = self.ConvertBezier2AirfoilCoordinates(bezier_thickness_x,
+                                                                                       bezier_thickness,
+                                                                                       bezier_camber_x,
+                                                                                       bezier_camber)
+
+            #Need to create interpolation of upper and lower surfaces to ensure we take data from same x-coordinates
+            interpolated_upper_surface_data = interpolate.CubicSpline(np.flip(self.reference_data[:self.idx_LE + 1, 0]), 
```

Ruff (0.8.2)

Local variable `interpolated_upper_surface_data` is assigned to but never used

Remove assignment to unused variable `interpolated_upper_surface_data`

(F841)